modules/developers-guide/pages/tutorials/counter-tutorial.adoc

```diff
@@ -102,23 +101,23 @@ Let's take a closer look at this sample program:
 * You can see that the `+currentValue+` variable declaration in this example includes the `+stable+` keyword to indicate the state—the value that can be set, incremented, and retrieved—persists.
 +
 This keyword ensures that the value for the variable is unchanged when the program is upgraded. 
-* The declaration for the `+currentValue+` variable also specifies that its type is a natural (`+Nat+`) number.
-* The program includes two public update methods—the `+increment+` and `+set+` functions—and one a query method, in this case, the `+get+` function.
+* The declaration for the `+currentValue+` variable also specifies that its type is a natural number (`+Nat+`).
+* The program includes two public update methods—the `+increment+` and `+set+` functions—and one a query method-the `+get+` function.
 --
 +
 For more information about stable and flexible variables, see link:../../language-guide/upgrades{outfilesuffix}[Stable variables and upgrade methods] in the link:../../language-guide/motoko{outfilesuffix}[_{proglang} Programming Language Guide_].
 +
-For more information about the differences between a query and an update, see link:../concepts/canisters-code{outfilesuffix}#query-calls[query calls] in link:../concepts/canisters-code{outfilesuffix}#canister-state[Canisters include both program and state].
+For more information about the differences between a query and an update, see link:../concepts/canisters-code{outfilesuffix}#query-update[Query and update methods] in link:../concepts/canisters-code{outfilesuffix}#canister-state[Canisters include both program and state].
 . Save your changes and close the file to continue.
 
-== Start the local network
+== Start the local {IC} blockchain
 
-Before you can build the `+my_counter+` project, you need to connect to the {IC} network either running locally in your development environment or running remotely on a subnet that you can access.
+Before you can build the `+my_counter+` project, you need to either connect to a local {IC} blockchain, running in your development environment, or to the {IC} blockchain mainnet. 
```

Comment:
Please adapt accordingly.

Reply:
I suggested something and will thus close this as resolved.